.github/CONTRIBUTING.md

```diff
@@ -216,3 +222,24 @@ from Transifex to the source code and vice versa.
 ### Translations (for translators) ###
 
 Please have a look at the specific [FAQ for translators](https://github.com/Cockatrice/Cockatrice/wiki/Translation-FAQ).
+
+
+# Release Management #
+
+### ToDo list around new main releases ###
+
+ - write a `Release Draft` on github with all changes since last main release
+ - build newest strings and upload them to transifex
+ - give translators some time --> pull newest translations from transifex and PR them back in
+ - **Final commit:** push a PR which adds the `Release Name` and `Version Number` in the `About` window
```

Comment:
I don't know of any alternative or automation here... maybe for the version number something is possible because we are going to have them in the release tag on github at some point?